pandas/core/arrays/boolean.py

```diff
@@ -327,29 +327,81 @@ def __getitem__(self, item):
 
         return type(self)(self._data[item], self._mask[item])
 
-    def _coerce_to_ndarray(self, dtype=None, na_value: "Scalar" = libmissing.NA):
+    def to_numpy(
+        self, dtype=None, copy=False, na_value: "Scalar" = lib._no_default, **kwargs
```

Comment:
kwargs here?

Reply:
The kwargs here should indeed not be necessary I think